plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -0,0 +1,945 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024, Cisco Systems
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+__author__ = ("A Mohamed Rafeek, Natarajan, Madhan Sankaranarayanan, Abhishek Maheshwari")
+
+DOCUMENTATION = r"""
+---
+module: accesspoint_workflow_manager
+short_description: accesspoint_workflow_manager used to automate bulk AP configuration changes.
+description:
+  - Automates bulk configuration changes for Access Point (APs).
+  - Modify AP display names, AP names, or other parameters.
+  - Filter specific device details, such as selecting devices with hostnames matching "NFW-AP1-9130AXE"
+  - Compares input details with current AP configurations and applies desired changes only to relevant APs.
+
+version_added: '6.6.0'
+extends_documentation_fragment:
+  - cisco.dnac.accesspoint_workflow_manager
+author: 
+    - A Mohamed Rafeek (@mohamedrafeek)
+    - Natarajan (@natarajan)
+    - Madhan Sankaranarayanan (@madhansansel)
+    - Abhishek Maheshwari (@abmahesh)
+       
+options:
+    config_verify:
+        description: Set to True to verify the Cisco Catalyst Center config after applying the playbook config.
+    type: bool
+    default: False
+    state:
+        description: The state of Cisco Catalyst Center after module completion.
+    type: str
+    choices: [ merged ]
+    default: merged
+  config:
+    description: List of details of AP being managed
+    type: list
+    elements: dict
+    required: True
+    device_fields:
+        description:
+            - Optional field to specify specific fields to display from the device details.
+            - Fields should be separated by commas without spaces before or after the commas.
+            - If not defined, all device fields are shown by default.
+            - Example: "id,hostname,family,type,mac_address,management_ip_address,ap_ethernet_mac_address,last_updated,up_time"
+        type: str
+        required: False
+    ap_selected_field:
+        description: Optional field to specify specific parameters from the Access Point configuration details.
+            Each field should be separated by commas without spaces.
+            If not defined, all fields will be shown by default.
+            Example: "mac_address,eth_mac,ap_name,led_brightness_level,led_status,location"
```

Comment:
Do we have any documentation for it? How does the customer know which ap_selected_field they can provide and what these fields are? Also, we use device_fields, but we also use ap_selected_field. What is the distinction between referring to it as device field (Which device is this? Is it not ap?) and ap_selected_field? Also any reason for not separating by space after comma? 

Is it field or fields?